src/FieldEncryptMapPluginInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Core\Block\FieldEncryptMapPluginInterface.
+ */
+
+namespace Drupal\field_encrypt;
+
+/**
+ * This interface describes how field_encrypt plugins should be structured.
+ *
+ * @package Drupal\field_encrypt
+ */
+interface FieldEncryptMapPluginInterface {
+
+  /**
+   * @return array An array of field types that contains an array of field values and associated encryption services.
+   *
+   * See \Drupal\field_encrypt\Plugin\FieldEncryptMap\CoreStrings for an example.
+   */
+  public function getMap();
```

Comment:
@nerdstein, 
I don't think so. I think that would be a service and the `encryption` service as an example already does this. I don't see a reason to create separate services / plugins to do what an encryption service should do. For example, an `file_encryption.encrypt` service should provide standard `encrypt()` and `decrypt()` functions.